conda/models/channel.py

```diff
@@ -1,5 +1,15 @@
 # Copyright (C) 2012 Anaconda, Inc
 # SPDX-License-Identifier: BSD-3-Clause
+"""Defines Channel and MultiChannel objects and other channel-related functions.
+
+Object inheritance:
+ ┌─────────┐
```

Comment:
Where's the meta-class? 😏 Just kidding. Looks good 👌.

Reply:
ah i forgot about that! do we think they're valuable at all? if so i'll move them into a comment below the docstring

cc @jaimergp

Reply:
see Jaime's comment below, suggesting to use mermaid for this, I know there are a few Sphinx extensions that can render `.. mermaid::` blocks and there is even a `autoclasstree` directive in https://github.com/mgaitan/sphinxcontrib-mermaid! Demo: https://sphinxcontrib-mermaid-demo.readthedocs.io/

Reply:
switched to `sphinxcontrib-mermaid`

Reply:
Nice!

I think we still need to add the requirement to https://github.com/conda/conda/blob/main/docs/requirements.txt and the package name to https://github.com/conda/conda/blob/main/docs/source/conf.py#L41

@kenodegard

Reply:
agreed, I'm also experimenting with adding sphinx-autoapi in #12798 ATM